mariner/mariner-2.x/mariner-2.0-hpc/install_utils.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+set -ex
+
+# Setup Mariner Extended packages repo
+curl -s -L https://packages.microsoft.com/cbl-mariner/2.0/prod/extended/x86_64/config.repo  | tee /etc/yum.repos.d/mariner-extended-prod.repo
+
+# Setup microsoft packages repository for moby
+# Download the repository configuration package
+# curl -s -L https://packages.microsoft.com/config/rhel/8/prod.repo | tee /etc/yum.repos.d/microsoft-prod.repo
+
+tdnf repolist
+
+# Install Kernel dependencies
+tdnf install -y kernel-headers-$(uname -r) \
```

Comment:
This is a case where I thought you might use $KERNEL (though i don't know if it's the same format)